src/main/java/com/iexec/core/sms/SmsService.java

```diff
@@ -0,0 +1,22 @@
+package com.iexec.core.sms;
+
+import com.iexec.common.utils.BytesUtils;
+import com.iexec.core.feign.SmsClientWrapper;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SmsService {
```

Comment:
Probably SmsService and SmsClientWrapper can be merged.

Reply:
👍